src/ucp/core/ucp_worker.c

```diff
@@ -804,52 +823,47 @@ static ucs_status_t ucp_worker_select_best_ifaces(ucp_worker_h worker,
 {
     ucp_context_h context = worker->context;
     uint64_t tl_bitmap    = 0;
-    ucp_worker_iface_t *ifaces;
+    ucp_rsc_index_t repl_ifaces[context->num_tls];
```

Comment:
this is gcc extension, let's use ucs_alloca

Reply:
AFAIK, variable length array was added in C99